packages/telemetry/src/context-aware-slog.js

```diff
@@ -356,7 +356,11 @@ export const makeContextualSlogProcessor = (
       // eslint-disable-next-line no-restricted-syntax
       case SLOG_TYPES.COSMIC_SWINGSET.RUN.FINISH: {
         assert(!!triggerContext);
-        persistContext(finalBody.remainingBeans ? {} : triggerContext);
+        persistContext(
+          finalBody.remainingBeans && finalBody.remainingBeans > 0
```

Comment:
I sometimes dislike what types force us to do. The `finalBody.remainingBeans &&` part is totally unnecessary at runtime.